js/project-list-item.js

```diff
@@ -36,8 +36,17 @@ var ProjectListItem = {
     var html = ProjectListItem.buildHTML(projectData);
     var $listItem = $(html);
 
+    $listItem.find(".star").on("click", function(event) {
+      event.preventDefault();
+      FavouritesManager.toggleProjectFavState(id,projectData.Title);
+
     $listItem.find(".project-title").on("click", ProjectListItem.viewProjectLinkClickHandler);
```

Comment:
some copying and pasting mistake here? `$listItem.find(".project-title").on("click", ProjectListItem.viewProjectLinkClickHandler);` should not be part of this clicking handler 😄